src/Analysis/Ast/Impl/Analyzer/Handlers/AssignmentHandler.cs

```diff
@@ -44,10 +45,10 @@ internal sealed class AssignmentHandler : StatementHandler {
                 value = Eval.UnknownType;
             }
 
-            if (node.Left.FirstOrDefault() is TupleExpression lhs) {
+            if (node.Left.FirstOrDefault() is SequenceExpression seq) {
```

Comment:
What happens if you have several complicated expressions on the left side of an assignment?
I think you can't look only at node.Left.First()
((x, r), y, z) = (1, 1), '', ''